apps/yggdrasil/priv/repo/migrations/20160303191523_create_users_roles.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Yggdrasil.Repo.Migrations.CreateUsersRoles do
+  use Ecto.Migration
+
+  def change do
+    create table(:users_roles, primary_key: false) do
+      add :user_id, references(:users, on_delete: :nothing)
+      add :role_id, references(:roles, on_delete: :nothing)
+
+      timestamps
+    end
+    create index(:users_roles, [:user_id])
```

Comment:
added a note above, but this gives me a more direct question to research. So the thought is the DB should be smart enough to use the index if supplied the first column or better put for example supplied in order.